app/views/ThreadMessagesView/index.tsx

```diff
@@ -237,7 +237,7 @@ class ThreadMessagesView extends React.Component<IThreadMessagesViewProps, IThre
 		const { subscription } = this.state;
 		// if there's no subscription, manage data on this.state.messages
 		// note: sync will never be called without subscription
-		if (!subscription) {
+		if (!subscription._raw) {
```

Comment:
Same as the other PRs.
You should have a better way to test this without looking for WM internals.